examples/apache-directory-list-local/webdav.conf

```diff
@@ -13,4 +13,15 @@
         Options +Indexes
         HeaderName /webdav-js/examples/apache-directory-list-local/header.html
     </Location>
+
+    # Add security
+    <Directory /var/www/webdav-js>
+        Options +Indexes
+        DAV On
+
+        Satisfy all
+        <LimitExcept GET HEAD OPTIONS PROPFIND>
```

Comment:
Definitely useful! I think we can probably even just limit to `GET`.

Reply:
Will need to test that

Reply:
...works 
https://github.com/dom111/webdav-js/pull/21/commits/15ebce33cd081579946cfab8e1144c644c9a79ce done